src/scripts/selectors/PlayerSelectors.js

```diff
@@ -33,26 +33,30 @@ export const getPlayingVideo: (
   }
 )
 
+export const getFormattedDuration: (
+  state: RootState
+) => string = createSelector(
+  [getPlayingVideo],
+  (playingVideo: ?VideoRecord): string => {
+    const duration: ?string = playingVideo && playingVideo.get('duration')
+    return formatDuration(duration)
+  }
+)
+
```

Comment:
Moved this function up in scope so that `getFormattedCurrentTime` could access it